package.json

```diff
@@ -134,7 +129,12 @@
     "nyc": "^15.1.0",
     "replace-in-file": "^6.1.0",
     "vue-eslint-parser": "^7.1.1",
-    "webpack-bundle-analyzer": "^3.9.0"
+    "webpack-bundle-analyzer": "^3.9.0",
+    "eslint-plugin-promise": "^5.1.0",
+    "stylelint": "^14.6.1",
+    "stylelint-declaration-strict-value": "^1.8.0",
+    "stylelint-order": "^5.0.0",
+    "stylelint-webpack-plugin": "^3.2.0"
   },
   "browserslist": [
     "ie >= 10"
```

Comment:
@Tom I think this slows things/adds up to the build size a lot. But BC...

Reply:
These dependencies were already here. I moved them to devDependencies so it reduces the build size unless you're actually debugging apostrophe core itself in which case they belong.

(Project level might depend on all of these anyway, but then again it might not, or it might want different major versions, so this can still be a reduction.)

Reply:
I'm sorry, I meant this part `"ie >= 10"`

Reply:
Oh, I see.

I'm not sure if this is or was doing anything.

We don't ship es5 support with apostrophe core anymore, it's in a separate module. I don't know how exactly babel decides which package.json file to look for this setting in. But it's probably looking for a project level one, or not looking at all because we always set our targets this way in our ES5 stuff anyway:

```
                      targets: {
                        browsers: '> 1%, IE 11, not dead'
                      }
```

It's a separate issue unrelated to this PR, but you could do a PR to remove it, and that would pass through QA normally. I suspect it won't have an impact but dead code is bad to have.

Reply:
I need to research it.